processes/spell-sheet-review.md

```diff
@@ -0,0 +1,17 @@
+# Spell Coordination Sheet Review
+
+## Checklist
+
+- [ ] Are all fields filled in?
+- [ ] Do contents match the discussion from the GovOps Meeting?
+    - [ ] Confirm all items were discussed at GovOps.
+    - [ ] Confirm no items from the discussion were omitted.
+- [ ] Are incomplete items (awaiting final figures) marked with a clear `TODO`?
+- [ ] If there is a Spark Spell:
+    - [ ] Is the proxy spell address listed in a "Trigger Spark Proxy Spell at" line?
+    - [ ] Are Spark Spell contents seperated ([example](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=684547605&range=J26))?
```

Comment:
Our only suggestion is to consider making this more generic to subDAOs in general, rather than exclusively Spark.

Reply:
Thank you, ldr.